CONTRIBUTING.md

```diff
@@ -194,11 +198,37 @@ Code Formatter] installed, you can apply formatting directly in IntelliJ.
 
 Note that only some sub-projects in the Elasticsearch project are currently
 fully-formatted. You can see a list of project that **are not**
-automatically formatted in [gradle/formatting.gradle](gradle/formatting.gradle).
+automatically formatted in
+[build-tools-internal/src/main/groovy/elasticsearch.formatting.gradle](build-tools-internal/src/main/groovy/elasticsearch.formatting.gradle).
+
+#### Imports
+
+It is also important that you configure the correct import ordering in
+IntelliJ.  We configure Spotless (above) to sort the imports in Java files,
+and while the [Eclipse Code Formatter] plugin supports an option for
+ordering imports, in practice this option appears unreliable, so instead
+you must configure IntelliJ's import ordering to match Spotless.
+
+   1. Open **Preferences > Editor > Code Style > Java**
+   2. Click the **Imports** tab
+   3. Set "Class count to use import with '*'" to 9999.
```

Comment:
The class count bit should be done by the config in `elasticsearch.ide.gradle` on import.